platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -170,17 +202,50 @@ - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshot
 #if TARGET_OS_IPHONE
                     UIImage *logoImage = [UIImage imageNamed:@"mapbox" inBundle:[NSBundle mgl_frameworkBundle] compatibleWithTraitCollection:nil];
                     
+                    
                     UIGraphicsBeginImageContextWithOptions(mglImage.size, NO, self.options.scale);
                     
                     [mglImage drawInRect:CGRectMake(0, 0, mglImage.size.width, mglImage.size.height)];
-                    [logoImage drawInRect:CGRectMake(MGLLogoImagePosition.x, mglImage.size.height - (MGLLogoImagePosition.y + logoImage.size.height), logoImage.size.width,logoImage.size.height)];
+                    
+                    CGRect logoImageRect = CGRectMake(MGLLogoImagePosition.x, mglImage.size.height - (MGLLogoImagePosition.y + logoImage.size.height), logoImage.size.width,logoImage.size.height);
+                    [logoImage drawInRect:logoImageRect];
+                    
+                    [[self attributionBackgroundView:CGRectMake(MGLLogoImagePosition.x + 10,
+                                                                logoImageRect.origin.y,
+                                                                attributionBackgroungWidth,
+                                                                attributionBackgroungHeight)].layer renderInContext:UIGraphicsGetCurrentContext()];
+                    
+                    int x = logoImage.size.width + 20;
```

Comment:
The macOS SDK positions the attribution view toward the bottom-left corner of MGLMapView, to the right of the logo view, because the zooming and rotation controls are located in the bottom-right corner by default. But there aren’t any zooming or rotation controls in snapshots, so it would be more consistent with other platforms to place the attribution at the bottom-right corner.